docs/jax-101.rst

```diff
@@ -0,0 +1,21 @@
+.. _jax-101:
+
+JAX 101
+=======
+
+These tutorials cover basic usage of JAX and its features, including some of the
+internal mechanisms that make JAX work. They start with the fundamentals and are
+meant to be read sequentially. For more in-depth discussions of JAX's design and
+implementation, in no particular order, see the :doc:`advanced guides <advanced_guides>`.
+
+.. toctree::
+   :maxdepth: 1
+
+   jit-compilation
+   automatic-vectorization
+   automatic-differentiation
+   working-with-pytrees
+   random-numbers
+   control-flow
+   stateful-computations
+   sharded-computation
```

Comment:
One more thing I missed (thanks to Yash for spotting). Could you move this to after `random-numbers`? `sharded-computation` is "Intro to Parallel programming" in the doc.